app/views/pages/new.slim

```diff
@@ -11,5 +11,9 @@ h2 = t('.title')
     = f.select :liquid_layout_id, LiquidLayout.all.map{|ll| [ll.title, ll.id] }, {}, class: "form-control"
 
   .form-group
+    = f.label :language_id, t('pages.edit.language_label')
+    = f.select :language_id, Language.all.map { |lang| [lang.name, lang.id] }, {}, class: "form-control"
```

Comment:
why did we end up needing to set the language on the create step? ak integration?